LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Should we adopt the Source Available License and make this effectively "enterprise-only"?